app/views/state_file/questions/id_sales_use_tax/edit.html.erb

```diff
@@ -11,7 +11,6 @@
           <%=
             f.cfa_radio_set(
               :has_unpaid_sales_use_tax,
-              label_text: t('.unpaid_sales_use_tax_label_html'),
               collection: [
```

Comment:
why was this taken out?

Reply:
there is not supposed to be a copy above this Yes/No question -- according to figma. I had included the same text twice & noticed it was different from the figma, so I removed it